### PR TITLE
[DEMO] Simple extension via ArgumentValueChanger for Pandas inplace argument

### PR DESCRIPTION
**Pull request by jaiparera**
Test cases passing
image

Mutations being killed
image

### CODE REVIEW

File: example/df.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Returns:
+    pd.DataFrame: A new DataFrame with the column dropped.
+    """
+    return df.drop(columns=[column_name], inplace=False)
+
+
+def fillna_inplace(df, value):
+    return df.fillna(value, inplace=True)
+
+def fillna_copy(df, value):
```
Comment (njhuey) [inline example/df.py:33]: mypy compliance?

**Review by njhuey — APPROVED** [review]
LGTM!

**Comment by dillonago** [comment]
Looks good to me!

File: example/df.py (hunk 1 above)
Comment (jaiparera) [reply example/df.py:33]: Push to next year

**Pull request merged**